Source/Pages/Gallery/LibraryMediaManager.swift

```diff
@@ -183,10 +183,12 @@ public class LibraryMediaManager {
                     }
                 }
 
-                do {
-                    try videoCompositionTrack.insertTimeRange(trackTimeRange, of: videoTrack, at: CMTime.zero)
-                } catch {
-                    ypLog("Unexpected error: \(error).")
+                if !YPImagePickerConfiguration.shared.library.allowPhotoAndVideoSelection {
+                    do {
+                        try videoCompositionTrack.insertTimeRange(trackTimeRange, of: videoTrack, at: CMTime.zero)
+                    } catch {
+                        ypLog("Unexpected error: \(error).")
+                    }
```

Comment:
What was the reason behind this?

Reply:
This change doesn't seem necessary, I think it came from the WIP branch but it's not strictly necessary for multi video selection when I tested without this change. I removed it here: https://github.com/rewardStyle/YPImagePicker/pull/27/commits/d11a88a5c675d3787e42e42e08037e91bba1cb99 @MikkoKuivanenRS